MastodonSDK/Sources/MastodonCommon/Preference/Preference+Notification.swift

```diff
@@ -38,6 +39,17 @@ extension UserDefaults {
         setNotificationCountWithAccessToken(accessToken: accessToken, value: count + 1)
     }
     
+    @objc public func getLastSelectedNotificationsTab(accessToken: String) -> Int {
+        let prefix = UserDefaults.notificationsLastTabIndexKeyPrefix
+        let key = UserDefaults.deriveKey(from: accessToken, prefix: prefix)
```

Comment:
Using `deriveKey` is clever here! I assume, the `accessToken` is permament in a sense that it doesn't change frequently? Just to make sure, that `lastSelectedNotificationsTab`-entries won't pile up in UserDefaults

Reply:
@zeitschlag from what I've seen it's long lived and I haven't seen a refresh being made. Also this technique is used for other defaults as well (which probably isn't a perfect excuse 🥸)

Reply:
It's a good enough excude 🤓